simulation/CMakeLists.txt

```diff
@@ -4,7 +4,12 @@ set(SUBSYS_DEPS common io surface kdtree features search octree visualization fi
 
 set(build FALSE)
 find_package(OpenGL)
-find_package(GLEW)
+if(APPLE)
+  # glew in homebrew installs glew-config.cmake
+  find_package(glew)
```

Comment:
```suggestion
  find_package(glew CONFIG)
```

Reply:
It uses the findglew file first atm:
https://dev.azure.com/PointCloudLibrary/pcl/_build/results?buildId=17217&view=logs&j=cd6332f9-d180-53f5-8d9a-3b5d9c3252a2&t=85d62bd5-4d0c-5e0e-700b-d959df91e277&l=101

Reply:
Alternatively just use the the original? The FindGLEW.cmake, finds glew with config search internally.